index.html

```diff
@@ -360,6 +391,45 @@ <h4>Octane</h4>
         </div>
       </section>
 
+      <section aria-labelledby="section-default-args">
```

Comment:
This section is just moved from its previous (incorrect) location under Templates